src/backend/distributed/metadata/dependency.c

```diff
@@ -315,6 +377,11 @@ SupportedDependencyByCitus(const ObjectAddress *address)
 				return true;
 			}
 
+			case OCLASS_EXTENSION:
```

Comment:
This is wrong place to add. We should add outside of `if (!EnableDependencyCreation)`  below switch

Reply:
Done in 876af1603cbed2741240088ce6418d3f26a5e215